modules/core/arrow-core-data/src/test/kotlin/arrow/core/OptionTest.kt

```diff
@@ -49,7 +50,8 @@ class OptionTest : UnitSpec() {
       TraverseFilterLaws.laws(Option.traverseFilter(), Option.applicative(), ::Some, Eq.any()),
       MonadFilterLaws.laws(Option.monadFilter(), ::Some, Eq.any()),
       HashLaws.laws(Option.hash(Int.hash()), Option.eq(Int.eq())) { it.some() },
-      SemigroupalLaws.laws(Option.semigroupal(), ::Some, ::bijection, associativeSemigroupalEq)
+      SemigroupalLaws.laws(Option.semigroupal(), ::Some, ::bijection, associativeSemigroupalEq),
```

Comment:
SemigroupalLaws shouldn't be necessary anymore, because they're included in MonoidalLaws.

Reply:
I have not included them yet but I could do something like `SemigroupalLaw.laws + listOf(..)` in `MonoidalLaws` if you want.

Reply:
Yeah, that works!